wiki/Gameplay/Game_modifier/xK/de.md

```diff
@@ -0,0 +1,50 @@
+---
+stub: true
+tags:
+  - xK
+  - 1K
+  - 2K
+  - 3K
+  - 4K
+  - 5K
+  - 6K
+  - 7K
+  - 8K
+  - 9K
+  - mod
+  - game modifier
+  - Spielmodifikation
+---
+
+# xK (Mod)
+
+*Hinweis: xK ist ein Sammelbegriff für die Tastenmodifikationen: 1K, 2K, 3K, 4K, 5K, 6K, 7K, 8K und 9K.*
+
+![1K Modsymbol](/wiki/shared/mods/1K.png "1K Modsymbol") ![2K Modsymbol](/wiki/shared/mods/2K.png "2K Modsymbol") ![3K Modsymbol](/wiki/shared/mods/3K.png "3K Modsymbol") ![4K Modsymbol](/wiki/shared/mods/4K.png "4K Modsymbol") ![5K Modsymbol](/wiki/shared/mods/5K.png "5K Modsymbol") ![6K Modsymbol](/wiki/shared/mods/6K.png "6K Modsymbol") ![7K Modsymbol](/wiki/shared/mods/7K.png "7K Modsymbol") ![8K Modsymbol](/wiki/shared/mods/8K.png "8K Modsymbol") ![9K Modsymbol](/wiki/shared/mods/9K.png "9K Modsymbol")
+
+*Für die vollständige Liste aller Mods, siehe: [Spielmodifikationen](/wiki/Game_modifier)*
+
+## Übersicht
+
+- Abkürzung: xK
+- Typ: Sonstige
+- Score-Multiplikator: 0,66x bis 1,00x
+- Standard-Tastenkürzel: `Z` (jeder Klick/Tastendruck geht von 4K zu 9K, dann von 1K bis 3K, bis die Mod deaktiviert wird)
+- Kompatible Spielmodi: ![][osu!mania]
+
+## Beschreibung
+
+*Hinweis: Alle xK-Mods werden auf osu!mania-spezifischen Beatmaps ignoriert. Die Verwendung der Mod 1K, 2K oder 3K sorgt dafür, dass das Spiel nicht gerankt wird.*
+
+Dies sind [Spielmodifikationen](/wiki/Game_modifier), die alle *konvertierten* [osu!mania](/wiki/Game_mode/osu!mania)-[Beatmaps](/wiki/Beatmap) dazu zwingen, die selektierte K-Einstellung zu verwenden. Die angenommene K-Einstellung der Beatmap wird abhängig von der K-Differenz zu Lasten des Score-Multiplikators ignoriert.
```

Comment:
I'm not really sure on the translation `k setting -> K-Einstellung` here, as well as in general the last phrase of this paragraph.